python/lsst/analysis/drp/colorColorFitPlot.py

```diff
@@ -106,6 +107,12 @@ class ColorColorFitPlotConfig(pipeBase.PipelineTaskConfig,
         optional=True,
     )
 
+    minPointsForFit = pexConfig.Field(
+        doc="Minimum number of valid objects to bother attempting a fit.",
+        dtype=int,
+        default=5,
```

Comment:
Does this need to be validated as > 0?

Reply:
Good idea.  I made it a `RangeField` and gave it `min=1`.